malaria24/ona/templates/ona/html_email.html

```diff
@@ -41,19 +41,19 @@
       <td colspan="4">5. Gender: <span class="variable">{{case.gender}}</span></td>
     </tr>
     <tr>
-      <td colspan="6">6. Reporting Health Facility: <span class="variable">{{case.last_name}}</span></td>
-      <td colspan="6">7. Province: <span class="variable">{{case.last_name}}</span></td>
+      <td colspan="6">6. Reporting Health Facility: <span class="variable">{{case.facility_name}}</span></td>
+      <td colspan="6">7. Province: <span class="variable">{{case.province}}</span></td>
     </tr>
     <tr>
-      <td colspan="6">8. Date notification form was sent/collected from health facility: <span class="variable">{{case.last_name}}</span></td>
-      <td colspan="6">9. Date notification form captured in MIS: <span class="variable">{{case.last_name}}</span></td>
+      <td colspan="6">8. Date notification form was sent/collected from health facility: <span class="variable">{{case.create_date_time|date:"D d M Y" }} {{case.create_date_time|date:"H:i"}}</span></td>
+      <td colspan="6">9. Date notification form captured in MIS: <span class="variable">{{case.created_at|date:"D d M Y" }} {{case.created_at|date:"H:i"}}</span></td>
     </tr>
```

Comment:
M24H-31 specified that they should be populated, I've now left 8 as it is (because we do have that info) but I've left 9 blank because we technically don't have an MIS yet.